### PR TITLE
[15.0][MIG] product: migration script

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Continues https://github.com/OCA/OpenUpgrade/pull/3283.

### CODE REVIEW

File: openupgrade_scripts/scripts/product/15.0.1.2/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        """
+        ALTER TABLE product_template_attribute_value
+        ADD COLUMN color integer
+        """,
+    )
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE product_template_attribute_value
+        SET color = FLOOR(random() * 11 + 1)::int
```
Comment (gdgellatly) [inline openupgrade_scripts/scripts/product/15.0.1.2/pre-migration.py:106]: Mimics upstream default

**Review by gdgellatly — APPROVED** [review]
LGTM

File: openupgrade_scripts/scripts/product/15.0.1.2/pre-migration.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply openupgrade_scripts/scripts/product/15.0.1.2/pre-migration.py:106]: Yeah, this is equivalent to `randint(1, 11)`

Comment (gdgellatly) [reply openupgrade_scripts/scripts/product/15.0.1.2/pre-migration.py:106]: @MiquelRForgeFlow Of course, just for other reviewers I find it helpful if you see something unusual to write that you checked it and it is correct. I do wonder for some of these things if it isn't worth a permalink to link to the upstream default function in a docstring or comment.

**Comment by gdgellatly** [comment]
@MiquelRForgeFlow When we do these replacement PR's I think we need some way to update the PR migration issue, manually or otherwise. Is there a command for this?

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**